From 1fb3367dcdcfb588bb3c12f5ccc86113f8af6a6b Mon Sep 17 00:00:00 2001
From: Jin Qian <jqian@aurora.tech>
Date: Mon, 2 Nov 2020 16:15:14 -0800
Subject: [PATCH 13/22] Disable JEMALLOC_TLS for QNX

TLS access triggers recurisive malloc during bootstrapping. Need to use
pthread_getspecific and pthread_setspecific with a follow up fix.

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
Change-Id: I0969f0afabb0a8a8e6b4cff8e5f9b109b01e5ed1
---
 configure.ac | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure.ac b/configure.ac
index c90f79f8..294a8038 100644
--- a/configure.ac
+++ b/configure.ac
@@ -755,6 +755,10 @@ case "${host}" in
 	  default_retain="1"
 	fi
 	;;
+  *-*-nto-qnx)
+	abi="elf"
+  force_tls="0"
+	;;
   *)
 	AC_MSG_RESULT([Unsupported operating system: ${host}])
 	abi="elf"
-- 
2.32.0

